src/components/layouts/Footer.tsx

```diff
@@ -0,0 +1,180 @@
+import { Flex, Link, Stack, Text } from '@chakra-ui/react';
+import { FC } from 'react';
+import NextLink from 'next/link';
+
+import {
+  DOCS_PAGE,
+  DOWNLOADS_PAGE,
+  GETH_DISCORD_URL,
+  GETH_REPO_URL,
+  GETH_TWITTER_URL
+} from '../../constants'
+
+import {
+  DiscordIcon,
+  GitHubIcon,
+  TwitterIcon
+} from '../UI/icons';
+
+export const Footer: FC = () => {
+  return (
+    <Flex
+      mt={4}
+      direction={{ base: 'column', lg: 'row'}}
+    >
+      <Flex
+        direction={{ base: 'column', md: 'row'}}
+        justifyContent={{md: 'space-between'}}
+        border='2px solid'
+        borderColor='brand.light.primary'
+      >
+        <Flex
+          sx={{ mt: '-2px !important' }}
+          borderBottom={{
+            base: '2px solid',
+            md: 'none'
+          }}
+          borderColor='brand.light.primary'
+        >
+          <Stack
```

Comment:
Why are we using a `Stack` here? I think the Stack is going to add some margins and make things a bit messy.

Suggestion:
Perhaps its more convenient to use a `Flex` OR...
1. remove the Stack since it is just wrapping 1 element (no need to have this wrapper)
2. apply those same styles to the `Link` component
3. make the `Link` as `display="flex"` & `height="full"`

IMO this should be applied to the rest of the links here. Simplifying everything a little bit.